qiskit_algorithms/gradients/base/base_sampler_gradient.py

```diff
@@ -73,10 +69,12 @@ def run(
                 ``circuits``. Defaults to None, which means that the gradients of all parameters in
                 each circuit are calculated. None in the sequence means that the gradients of all
                 parameters in the corresponding circuit are calculated.
-            options: Primitive backend runtime options used for circuit execution.
-                The order of priority is: options in ``run`` method > gradient's
-                default options > primitive's default setting.
-                Higher priority setting overrides lower priority setting
+            shots: Number of shots to be used by the underlying sampler. If a single integer is
+                provided, this number will be used for all circuits. If a sequence of integers is
+                provided, they will be used on a per-circuit basis. If none is provided, the
```

Comment:
I guess I have a similar question around these sentences like I had above. And here I see fidelity's but am missing seeing such an object

> If none is provided, the
> fidelity's default number of shots will be used for all circuits. If this number is
>  also set to None, the underlying primitive's default number of shots will be used
> for all circuits.

Reply:
I'll update the docstring when I have the confirmation on the other comment that my proposal for the docstring works for you!